CoreWidgetProvider/Helpers/GPUStats.cs

```diff
@@ -29,11 +29,14 @@ public class Data
 
     public GPUStats()
     {
-        AddGPUPerfCounters();
+        GetGPUs();
+        GetGPUPerfCounters();
     }
 
-    public void AddGPUPerfCounters()
+    public void GetGPUs()
```

Comment:
super nit: Suggesting alternative method names: `LoadGPUs()` or `Initialize()`. Caller of a `Get*` method might expect a return value.